### DISCUSSION TITLE
Clear the chart if no data

### DISCUSSION THREAD

**Discussion by matpascaud** [patrykandpatrick/vico#414] [Ideas]
**Describe the bug**
At some point, there is no data to display and would like to clear the whole chart without any label nor values on Y.
Maybe I am missing a function to clear all.

**Module**

- [ ] `compose`
- [X ] `views`
- [ ] `compose-m2`
- [ ] `compose-m3`
- [ ] `core` (select if none of the above apply)

**To reproduce**
Empty the chart entry model producer.
**Expected behavior**
I expect to have no remaining data. In my case, I still have marker with 0 value and the first label.

**Screenshots**
![screen01](https://github.com/patrykandpatrick/vico/assets/534304/03ed986c-e0d4-491e-8ed3-f4309edbd5e0)
![screen02](https://github.com/patrykandpatrick/vico/assets/534304/c78517f8-6ba0-4b7b-9ea6-7707b465ba02)

**Android versions**
Versions of Android where the bug occurs.

**Additional context**

**Comment by patrickmichalik** [comment]
Hello! Thanks for the suggestion. We’ll be looking into this.

**Comment by patrickmichalik** [comment]
[Vico 1.13.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v1.13.0-alpha.9) introduces chart placeholders. The default placeholder is blank, producing the result described under “Expected behavior.” Cheers!

**Reply by matpascaud** [reply]
Great. Thank you.

**Reply by patrickmichalik** [reply]
You’re welcome!

**Comment by patrickmichalik** [comment]
Vico 1.13 is now stable, so I’ll be closing this. Cheers!
